kolibri/plugins/app/utils.py

```diff
@@ -21,6 +23,19 @@ def register(self, **kwargs):
             if capability in kwargs:
                 self._capabilities[capability] = kwargs[capability]
 
+    def get_initialize_url(self, next_url=None):
+        if not self.enabled:
+            raise RuntimeError("App plugin is not enabled")
+        # Import here to prevent a circular import
+        from kolibri.core.device.models import DeviceAppKey
+
+        url = reverse(
+            "kolibri:kolibri.plugins.app:initialize", args=(DeviceAppKey.get_app_key(),)
```

Comment:
`get_app_key()` returns a UUID type, but `reverse` is expecting a string, so we need to convert when passing as an arg. I tried to push a fix for this myself but got a repo authentication error.

Reply:
Oh - that's odd. It should really always return a string. Will look into it.